soweego/target_selection/musicbrainz/common.py

```diff
@@ -0,0 +1,21 @@
+import os
+import requests
+
+def get_path():
+    """Return the path of the current module"""
+    path = os.path.abspath(__file__)
+    return os.path.dirname(path)
+
+def get_output_path():
```

Comment:
Please delegate this to your CLI main function, as _click_ will handle the process